sections/01_abstract.md

```diff
@@ -3,3 +3,14 @@
 Offspring of the [Deep Review](https://github.com/greenelab/deep-review).
 We cite the Deep Review like [@doi:10.1101/142760] or [@tag:deep_review].
 The source repository is [@url:https://github.com/greenelab/manubot-rootstock].
+
+Equations can be integrated and labeled like this `{{ '{{ \'{#eq:label}\' }}' }}` and then
+referenced using `{@eq:label}` like this for Equation @eq:line.
+
+$$ y = mx + b $$ {{ '{#eq:line}' }}
+
+Images can be included and labeled like `{{ '{{ \'{#fig:label}\' }}' }}` and then
+referenced using `{@fig:label}` like this for Figure @fig:lotus.
```

Comment:
> That's what I think too, but I wanted to cover the bases with supporting the more complicated before

Let's go with the simple method, as I think its universally preferable and makes fewer brackets which appear to be pain points. If we really want to make sure we cover the corner cases, we should implement test manuscripts, rather than put the non-preferred usage in the example.